data_model/src/transaction.rs

```diff
@@ -715,6 +699,12 @@ mod http {
             self
         }
 
+        /// Set executable for this transaction
+        pub fn with_executable(mut self, executable: Executable) -> Self {
```

Comment:
it's only used for testing

Reply:
It's not different from `with_wasm` or `with_instructions`.
Also doesn't break any invariants because we are working with `TransactionBuilder`, not `SignedTransaction`.

Reply:
oh, I see it's defined on `TransactionBuilder`. But we already have `with_wasm` and `with_instructions`. I think you can remove both in favor of the `with_executable(mut self, executable: impl Into<Executable>`

Reply:
good point

Reply:
it won't work unfortunately with `with_instructions` because it's more involved than just call `into`